### ISSUE TITLE
pause song with space key moves position in song list

### ISSUE THREAD

**Issue by rafaelleru** [rafaelleru/torrent-player#34]
see #33

**Comment by niralpokal** [comment]
I can make prevent the event from bubbling, so that the space bar will only pause and start? Or do you want to move all the key event listeners to a parent component and pass the values down? We would also need to make a slight change to the textbox to also prevent the input event from bubbling. Let me know your thoughts?

**Comment by rafaelleru** [comment]
Reading the electron docs it seems like the key events should be in the main process but I'm open to other solutions .

https://electron.atom.io/docs/tutorial/keyboard-shortcuts/

**Comment by niralpokal** [comment]
That makes sense, I can see if i can make the changes. I might not be able to make them today but I will try.  Ill keep you updated!

**Comment by niralpokal** [comment]
I was able to make a simple change that fixes this issue by capturing the event. If you want to use the global shortcuts for electron, I think we will need to reorganize the components.  I can create a pull request tomorrow and we can review if you like the changes or not. If you dont like it, we can maybe then hash out a plan to use main process and global shortcuts. :)

**Comment by rafaelleru** [comment]
Yeah o think that use electron shortcuts would be better in terms of standardization and best practices. I think that use this would be better.

**Comment by niralpokal** [comment]
It appears that the global shortcuts apply even when the app isnt focused. So if a user presses the space bar when in a different app, it will pause the music. I can use the global shortcuts to only capture the media keys? Looking in the electron doc that you linked, it even recommends to use normal event listeners in the app like we have already done.

**Comment by rafaelleru** [comment]
ok, so we can keep the actual solution and prevent the buubling. 👍

**Comment by niralpokal** [comment]
I'll push my changes today after work :)
